cromwell-backend/CHANGELOG.MD

```diff
@@ -0,0 +1,5 @@
+# Cromwell-Backend Change Log
+
+1.0:
+=====
+
```

Comment:
Are we planning on releasing cromwell-backend as a JAR by itself?  If not perhaps we should just roll this in to the top level changelog?  I must admit I'm not entirely sure what the deploy plan is for this subproject